src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -210,15 +232,22 @@ const GroupDetails = createReactClass({
     } else if (this.state.loading || !group) return <LoadingIndicator />;
 
     return (
-      <DocumentTitle title={this.getTitle()}>
-        <div className={this.props.className}>
-          <GroupHeader params={params} project={project} group={group} />
-          {React.cloneElement(this.props.children, {
-            group,
-            project,
-          })}
-        </div>
-      </DocumentTitle>
+      <Feature features={['sentry10']}>
+        {({hasFeature}) => (
+          <React.Fragment>
+            {hasFeature &&
+              showGlobalHeader && (
+                <GlobalSelectionHeader
+                  organization={organization}
+                  forceProject={project}
+                />
+              )}
+            {hasFeature &&
+              showGlobalHeader && <PageContent>{this.renderContent()}</PageContent>}
+            {!hasFeature || (!showGlobalHeader && this.renderContent())}
```

Comment:
Do you need the `!showGlobalHeader` here? Do we use GroupDetails without sentry10 and with the header?